examples/db-templates/postgresql-ephemeral-template.json

```diff
@@ -16,7 +16,8 @@
   },
   "message": "The following service(s) have been created in your project: ${DATABASE_SERVICE_NAME}.\n\n       Username: ${POSTGRESQL_USER}\n       Password: ${POSTGRESQL_PASSWORD}\n  Database Name: ${POSTGRESQL_DATABASE}\n Connection URL: postgresql://${DATABASE_SERVICE_NAME}:5432/\n\nFor more information about using this template, including OpenShift considerations, see https://github.com/sclorg/postgresql-container/blob/master/9.5.",
   "labels": {
-    "template": "postgresql-ephemeral-template"
+    "template": "postgresql-ephemeral-template",
+    "app": "postgresql-ephemeral-template"
```

Comment:
The app label should be parameterized so that when a DB template is used with another application and we want to associate it with that app we can.